mono_build_scripts_linux.sh

```
@@ -0,0 +1,20 @@
+    mkdir -p "$SOURCE_PATH"
+    dotnet nuget add source "$SOURCE_PATH" --name "$SOURCE_NAME"
+fi
+
+# Build editor binary
+scons compiledb=yes dev_build=yes platform=linuxbsd target=editor module_mono_enabled=yes
+
+# Generate glue sources
+bin/redot.linuxbsd.editor.dev.x86_64.mono --headless --generate-mono-glue modules/mono/glue
+
+#Generate binaries
+./modules/mono/build_scripts/build_assemblies.py --godot-output-dir=./bin --godot-platform=linuxbsd --push-nupkgs-local "~/MyLocalNugetSource/"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)